contracts/Users.sol

```diff
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.0;
-
+import "@openzeppelin/contracts/access/Ownable.sol";
```

Comment:
great, but you don't inherit `Ownable` contract => compilation error because `onlyOwner` is unknown.

Reply:
done. my bad that the commits were not pushed